doc/developingCubeBrowsers/interactions_2.ipynb

```diff
@@ -0,0 +1,390 @@
+    "def pick_cube(sender):\n",
+    "    cube = iris.load_cube(files.value, plots.value)\n",
+    "    print cube\n",
+    "    return cube\n",
+    "load_button.on_click(pick_cube)\n",
+    "\n",
+    "# Next I need to offer some plot customizations.  \n",
+    "# The following is a list of widgets which can be used to choose customization options.\n",
+    "colors = ipywidgets.Dropdown(\n",
+    "    options=['None', 'blue', 'green', 'red', 'cyan', 'magenta', 'yellow', 'black', 'white'],\n",
+    "    value='blue',\n",
+    "    description='colors')\n",
+    "cmap = ipywidgets.Dropdown(\n",
+    "    options=['None', 'viridis', 'inferno', 'plasma', 'magma', 'jet', 'summer', 'autumn'],\n",
+    "    value='None',\n",
+    "    description='cmap')\n",
+    "norm = ipywidgets.Dropdown(\n",
+    "    options=['None', 'Autoscale(A)', 'Autoscale_None(A)', 'scaled()'],\n",
+    "    value='None',\n",
+    "    description='norm')\n",
+    "levels = ipywidgets.Text(\n",
+    "    description='levels',\n",
+    "    value='np.linspace(np.min(cube.data), np.max(cube.data), 10)')\n",
+    "origin = ipywidgets.Dropdown(\n",
+    "    options=['None', 'upper', 'lower', 'image'],\n",
+    "    value='None',\n",
+    "    description='origin')\n",
+    "extend = ipywidgets.Dropdown(\n",
+    "    options=['neither', 'both', 'min', 'max'],\n",
+    "    value='both',\n",
+    "    description='extend')\n",
+    "linestyles = ipywidgets.Dropdown(\n",
+    "    options=['None', 'solid', 'dashed', 'dashdot', 'dotted'],\n",
+    "    value='None',\n",
+    "    description='linestyle')\n",
+    "\n",
+    "formatting = [colors, cmap, norm, levels, origin, extend, linestyles]\n",
+    "\n",
+    "# This button will prompt the appearance of the customization options above.\n",
+    "confirm_button = ipywidgets.Button(description='Customize')\n",
+    "def custom_options(sender):\n",
+    "    for i in range(len(formatting)):\n",
+    "        IPython.display.display(formatting[i])    \n",
+    "confirm_button.on_click(custom_options)\n",
+    "\n",
+    "# container is a box with all the selectors I want to display in it.\n",
+    "container1 = ipywidgets.Box(children=[files, plots])\n",
+    "container2 = ipywidgets.Box(children=[load_button, confirm_button])\n",
+    "IPython.display.display(container1, container2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "\n"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 2",
+   "language": "python",
+   "name": "python2"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.11"
+  },
+  "widgets": {
```

Comment:
can you remove all of these from the notebook before you save it, to reduce our source overhead?